paperless-ngx/config.yaml

```diff
@@ -37,6 +37,10 @@ options:
   keyfile: privkey.pem
   ingress_auth: false
   tika_gotenberg: false
+  backup_enabled: false
+  backup_path: /share/paperless/exports
```

Comment:
You must avoid absolute path since it's a docker container. 

If files aren't dropped into /share/ it will be lost in the void of the running instance file system